modules/loader-utils/test/zip-utils/data-view-provider.spec.ts

```diff
@@ -1,7 +1,7 @@
 import test from 'tape-promise/tape';
-import {DATA_ARRAY} from '../data/test.zip.js';
-import {DataViewFileProvider} from '../../src/lib/parsers/parse-zip/data-view-file-provider';
-import {signature} from '../../src/lib/parsers/parse-zip/local-file-header';
+import {DATA_ARRAY} from '@loaders.gl/i3s/test/data/test.zip.js';
```

Comment:
We normally don't include .js extension in imports.
```suggestion
import {DATA_ARRAY} from '@loaders.gl/i3s/test/data/test.zip';
```